script/copy-fixtures

```diff
@@ -4,10 +4,10 @@
 # be added to the Carthage workspace. This avoids confusing Xcode with
 # "subprojects" that are actually just originating from a fixture.
 
-destination="${BUILT_PRODUCTS_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}"
+destination="$2"
 
 for fixture in "carthage-fixtures-ReactiveCocoaLayout-master" "CartfileOnly" "CartfilePrivateOnly" "NoCartfile" "SchemeDiscoverySampleForCarthage-0.2" "Swell-0.5.0" "SampleMultipleSubprojects" "SampleGitSubmodule" "DependencyTest" "WorkspaceWithDependency"
 do
 	rm -Rf "$destination/$fixture"
-	unzip -q -d "$destination" "${SRCROOT}/Tests/CarthageKitTests/fixtures/$fixture"
+	unzip -q -d "$destination" "$1/$fixture"
```

Comment:
We should probably add existence checks to all these variables with `${foo:?}`.

Based on a quick reading of `man unzip`, I don't think our execution of it will overwrite, but redundant safety doesn't hurt…